### PR TITLE
This adds support for the Swift Package manager

### PR DESCRIPTION
**Pull request by mauricekraus**
To address the problem discussed in #32, I have added support for the Swift Package Manager.

### CODE REVIEW

**Comment by mauricekraus** [comment]
@fassko Is it possible to review/merge this one?

**Comment by fassko** [comment]
Thanks @mauricekraus I will try to do it in coming days.

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import PackageDescription
+
+let package = Package(
+    name: "RxAnimated",
+    platforms: [
+        .iOS(.v9),
+        .tvOS(.v10),
+    ],
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
```
Comment (fassko) [inline Package.swift]: Nit-picky but let's remove not needed comments in this file.

File: Sources/RxAnimated/RxAnimated+animations.swift [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import RxSwift
 import RxCocoa
+#if canImport(UIKit)
```
Comment (fassko) [inline Sources/RxAnimated/RxAnimated+animations.swift]: If in Package.swift we specify platforms do we need to do this `canImport(UIKit)` here?

File: Tests/RxAnimatedTests/RxAnimatedTests.swift [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import XCTest
```
Comment (fassko) [inline Tests/RxAnimatedTests/RxAnimatedTests.swift]: I think we can remove test files, because there are no tests for Linux platforms.

Comment (mauricekraus) [reply Tests/RxAnimatedTests/RxAnimatedTests.swift]: All tests in general or just the Linux test case? I think adding tests in the future would improve the library.

Comment (fassko) [reply Tests/RxAnimatedTests/RxAnimatedTests.swift]: Not all, only those which are template based.

**Comment by mauricekraus** [comment]
@fassko I resolved all your requests

**Comment by fassko** [comment]
Thanks @mauricekraus for your contribution! :)

**Comment by fassko** [comment]
🎉  Congrats

 🚀  RxAnimated (0.7.0) successfully published
 📅  March 6th, 21:37
 🌎  https://cocoapods.org/pods/RxAnimated
 👍  Tell your friends!

**Pull request merged**